scripts/src/java/org/oppia/android/scripts/common/RepositoryFile.kt

```diff
@@ -40,7 +40,7 @@ class RepositoryFile() {
         !isProhibited &&
           file.isFile &&
           file.name.endsWith(expectedExtension) &&
-          file.name !in exemptionsList
+          retrieveRelativeFilePath(file, repoPath) !in exemptionsList
```

Comment:
Maybe pull this into the PR where RepositoryFile was introduced for simplicity?

Reply:
Done in the regex-checks PR.